python/pynessie/utils/expression_util.py

```diff
@@ -1,20 +1,19 @@
-# -*- coding: utf-8 -*-
```

Comment:
this is still present in many more files while PEP-263 isnt needed for python3 at all, wdyt about removing them as well?

Reply:
Removed this comment from other files.